starter-code/routes/celebrities.routes.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express')
+const router = express.Router()
+const CelebrityModel = require('../models/celebrity')
+
+
+router.get('/celebrities', (req, res, next) => {
+    CelebrityModel.find()
+        .then((celebrity)=>{

```

Comment:
you are getting an array of several celebrities. So using the parameter name celebrities would make more sense.